### PR TITLE
Commerce 10850

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
Story https://issues.liferay.com/browse/COMMERCE-9392
Tech Task: https://issues.liferay.com/browse/COMMERCE-10850
Previous PR: ~~https://github.com/liferay-commerce/liferay-portal/pull/3465~~ -> https://github.com/liferay-commerce/liferay-portal/pull/3507
Forwarded PR: https://github.com/brianchandotcom/liferay-portal/pull/131740

### CODE REVIEW

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@MahmoudAzzam please check my comment, thanks!

**Comment by andrea-ale-sbarra** [comment]
ci:forward

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/CSDiagramEntryInfoItemFields.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.product.content.web.internal.info.item;
+package com.liferay.commerce.product.content.web.internal.info;
```
[the remark this answers is not in the document]
Comment (MahmoudAzzam) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/CSDiagramEntryInfoItemFields.java:15]: @alessiorendina Can you please clarify what is needed here?

**Comment by alessiorendina** [comment]
@MahmoudAzzam next time please close the PR when you forward it, thanks!

**Pull request closed without merging**